src/olympia/versions/models.py

```diff
@@ -645,6 +645,17 @@ def was_auto_approved(self):
         return False
 
 
+class VersionPreview(BasePreview, ModelBase):
+    version = models.ForeignKey(Version, related_name='previews')
```

Comment:
Depending on what we do in reviewer tools you might want to add a FK to `Addon` as well, to make sure reviewers see all previews that might someday end up being listed depending on what happens to each individual version ?

Reply:
Yeah we can link the preview back to the addon with `version.addon` if necessary, but I would have thought most of the time we'll be working on the Preview for a single version anyway.

The review style is like legacy addons so only after a version has been approved will it be available to be public - interim unreviewed versions will be disabled.

Reply:
Let's ignore this for now then.